### ISSUE TITLE
Model Convergence Issue

### ISSUE THREAD

**Issue by hhqweasd** [HengLan/CGSTVG#14]
I've noticed that your model requires the parameters of the text and resnet modules to be fixed. However, when training directly with the provided configuration file (the parameters of the text and resnet is not fixed), the model fails to converge. What could be the reason for this?

**Comment by GX77** [comment]
The parameters of the text and resnet modules need to be trained instead of being fixed. Are you using the default parameter configuration?

**Comment by hhqweasd** [comment]
@GX77 Using the default parameters will result in the model's failure to converge, with the loss not decreasing. Only after I froze the parameters of the text and ResNet was the training able to converge. However, the results were quite different from the original ones. The following are the reproduced results on hcstvg2:
gt_viou: 0.6050
tiou: 0.5738
viou: 0.3543
viou@0.3: 0.5701
gt_viou@0.3: 0.8492
viou@0.5: 0.2829
gt_viou@0.5: 0.6913

There were several inconsistent situations during training: 

1. I trained with 8 GPUs; 
2. My machine had no Internet connection. The pre-trained parameters for 2D, 3D, and text were all downloaded to the specified path in advance and then loaded in the code. 
3. The authors used two folder for the text model:
[self.body = RobertaModel.from_pretrained("model_zoo/roberta-base/")](https://github.com/HengLan/CGSTVG/blob/f062d84b72aee8be4425a4a5ab20aa5b077ca142/models/language_model/bert.py#L45)
[self.tokenizer = RobertaTokenizerFast.from_pretrained(pretrained_model_name_or_path='model_zoo/roberta/')](https://github.com/HengLan/CGSTVG/blob/f062d84b72aee8be4425a4a5ab20aa5b077ca142/models/language_model/bert.py#L46)
I simply copied roberta-base to roberta without modifying the codes. I'm not sure whether roberta-base==roberta in cgstvg.

Additionally, when using the test code to test on hcstvgv2, the performance when directly using the pre-trained parameters (hcstvg2.pth) was significantly lower than the published results:
gt_viou: 0.6086
tiou: 0.6031
viou: 0.3781
viou@0.3: 0.6155
gt_viou@0.3: 0.8466
viou@0.5: 0.3245
gt_viou@0.5: 0.6962

**Comment by GX77** [comment]
During training, the number of A100 GPUs is set to 16 in HCSTVGv1, 32 in HCSTVG-v2, and 64 in the VidSTG dataset. You can try to increase the batch size to 16. If the batch size is too small, training may not converge.
